### PR TITLE
Refactor code that set values to pix object in config.ru file

### PR DESCRIPTION
**Pull request by marciojg**
@pedrofurtado I did not understand how can test the method `generate_html_with(env)` 😅 What the format of param `env`? And How I can test rake locally? Can help me with this?  🙃

But, deducing that the method `params` returns a hash I created the PR 😇

### CODE REVIEW

File: demo/config.ru [hunk 1]
```diff
@@ -9,17 +9,13 @@ def generate_html_with(env)
   pix         = QrcodePixRuby::Payload.new
 
   unless qrcode_data.empty?
-    pix.pix_key        = qrcode_data['pix_key']           if !qrcode_data['pix_key'].nil?        && !qrcode_data['pix_key'].empty?
```
Comment (pedrofurtado) [inline demo/config.ru:12]: @marciojg Maybe we can write this snippet a little bit different, in order to keep `repeatable` working in form (if we delete it from hash, the respective form field appears empty for user 😞):

```ruby
qrcode_data.keys.each do |key|
  v = qrcode_data[key]
  pix.public_send("#{key}=", key == 'repeatable' ? v == 't' : v) if !v.nil? && !v.empty?
end
```

**Review by pedrofurtado — CHANGES_REQUESTED** [review]

File: demo/config.ru (hunk 1 above)
Comment (marciojg) [reply demo/config.ru:12]: Ok. I will run some tests with the Docker file that you create!

Comment (pedrofurtado) [reply demo/config.ru:12]: After this change, we are ready for merge 🎉  ✅

Comment (marciojg) [reply demo/config.ru:12]: @pedrofurtado I haven't been able to do it yet.

Soon I will do it. :)

**Comment by marciojg** [comment]
@pedrofurtado  Finally, I could send the commit :) 

Tks by de Dockerfile on the demo. It was very useful.

**Review by pedrofurtado — APPROVED** [review]

**Pull request merged**